src/home-page/components/CategoryCard.tsx

```diff
@@ -37,14 +38,9 @@ export const CategoryCard: React.FC<CardProps> = ({
   clickHandler,
 }) => {
   return (
-    <CategoryCardContainer
-      onClick={clickHandler}
-    >
-      <CardHeaderContainer
-        title={title}
-        subheader={description}
-      />
-      <AvatarContainer src={avatarSrc} backgroundColor={avatarBackground} />
+    <CategoryCardContainer onClick={clickHandler}>
+      <CardHeaderContainer title={title} subheader={description} />
+      <AvatarContainer src={avatarSrc} background={avatarBackground} />
```

Comment:
I think you could add redirection to the details screen already, casue the rooting is ready.

Reply:
Done.